app/models/tv_cast_member.rb

```diff
@@ -0,0 +1,29 @@
+class TVCastMember
```

Comment:
I renamed this file from `TVCast` to `TVCastMember` to make pluralization more instinctive.

The other changes are the same pattern for removing instance variables and instantiating the object.